### PR TITLE
Merge dev into cmake

### PR DESCRIPTION
**Pull request by AgoinsUSGS**
A built version with these changes can be located at /work/users/agoins/check_this_directory_out/ 

unit / cat / app tests are being ran and logged, don't merge this until Ethan runs through his tests on it.

### CODE REVIEW

File: isis/src/base/objs/BulletShapeModel/unitTest.cpp [hunk 1]
```diff
@@ -51,11 +51,14 @@ void testGroundPointToCamera(Latitude &lat,
...
                              BulletShapeModel &bulletModel);
 
-/** 
+/**
  * Unit test for Bullet Physics Ray Tracing library
- *  
+ *
  * @internal
  *   @history 2017-03-19 Kris Becker
+ *   @history 2018-08-07 Adam Goins - Removed qDebut() output of maxnumtriangles and maxnumparts
```
Comment (kaitlyndlee) [inline isis/src/base/objs/BulletShapeModel/unitTest.cpp]: Typo qDebut()

**Comment by kaitlyndlee** [comment]
Other than that, looks good!

**Comment by krlberry** [comment]
Since this is 15 days old, could you merge with dev again before pulling this into the cmake branch?

**Comment by AgoinsUSGS** [comment]
I just updated it with the current dev and cmake branches this should be ready to roll.

**Comment by SgStapleton** [comment]
Got the go-ahead to get this one merged.

**Pull request merged**